README.md

```diff
@@ -1 +1,62 @@
-# ToDoList-Review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Event Page
+
+> Todo List.
+
+![screenshot](https://github.com/miftah1991/ToDoList/blob/Milestone1-ListStructure/img/img-1.png)
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- ES 6
+- Webpack
+
+## Getting Started
+To get a local copy follow these simple steps.
+
+1.- Clone this project
+
+2.- Open the project directory in your terminal
+
+3.- In the terminal, install the project dependencies by running bellow command
+
+```
+npm install
+```
+4.- To see the home page open index.html with your browser.
+
+
+## Live Demo
+
+[Live Demo Link](https://miftah1991.github.io/Awesomebooks-ES6/)
+
+
+## Author
+
+👤 **Miftah Amin**
+
+- GitHub: [@miftah1991](https://github.com/miftah1991)
+- Twitter: [@miftah1991](https://twitter.com/miftah_amin1991)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/miftah1991/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+**📝 License**
+----------------------------------------------------------------------
+This project is MIT licensed.
+## Acknowledgments
+
+- [Cindy Shin](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Cindy%20Shin%20in%20Behance) in Behance for her [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```

Comment:
I think you can remove this line as this is just for the capstone project we did.